src/exchange-connector/exchangeDownloader.ts

```diff
@@ -12,38 +12,32 @@ import fetch, { Response } from "node-fetch";
 import path from "path";
 
 import { RestApi, FileInfo, Categories } from "./exchangeTypes";
+import { ramlToolLogger } from "../logger";
 
 const DEFAULT_DOWNLOAD_FOLDER = "download";
 const ANYPOINT_BASE_URI = "https://anypoint.mulesoft.com/exchange/api/v2";
+const ANYPOINT_BASE_URI_WITHOUT_VERSION =
+  "https://anypoint.mulesoft.com/exchange";
 
-export function downloadRestApi(
+export async function downloadRestApi(
   restApi: RestApi,
   destinationFolder: string = DEFAULT_DOWNLOAD_FOLDER
 ): Promise<void | Response> {
-  return new Promise((resolve, reject) => {
-    if (!restApi.fatRaml) {
-      reject(
-        new Error(
-          `Fat RAML download information for ${restApi.assetId} is missing`
-        )
-      );
-    }
-
-    ensureDirSync(destinationFolder);
+  if (!restApi.id) {
+    ramlToolLogger.info(
```

Comment:
should this be a warning?

Reply:
It might to be useful if the log message says RestApi ID is missing

Reply:
Yeah these are warnings. The only reason I made them infos is because with our current config warnings are not displayed.

Reply:
We are only checking the ID but other information is missing as well.